starwars/src/App.css

```diff
@@ -1,8 +1,19 @@
 .App {
   text-align: center;
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+ 
 }
+
 .Header {
   color: #443e3e;
   text-shadow: 1px 1px 5px #fff;
+  border: 1px solid red;
```

Comment:
Using Percentages or *rem/em* instead of *pixels* can help make the component more responsive. Specifically, it increases the *fluidity* to match the screen size.